index.html

```diff
@@ -919,11 +945,100 @@ <h4>Verify Proof (eddsa-jcs-2022)</h4>
             </dd>
           </dl>
 
-          <p>
-The algorithm is the same as the one described in Section
-[[[#verify-proof-eddsa-rdfc-2022]]] with the changes outlined in
-Section [[[#eddsa-jcs-2022]]].
-          </p>
+          <ol class="algorithm">
+            <li>
+Let |unsecuredDocument| be a copy of |securedDocument| with
+the `proof` value removed.
+            </li>
+            <li>
+Let |proofOptions| be the result of a copy of |securedDocument|.|proof| with `proofValue`
+removed.
+            </li>
+            <li>
+Let |proofBytes| be the
+<a data-cite="VC-DATA-INTEGRITY#multibase-0">Multibase decoded base58-btc
+value</a> in |securedDocument|.|proof|.|proofValue|.
+            </li>
+            <li>
+Perform the following checks  between the |securedDocument|.<var>@context</var> and
+the |proofOptions|.<var>@context</var>:
+              <ol class="algorithm">
+                <li>
+The |securedDocument|.<var>@context</var> MUST contain all values contained in the
+|proofOptions|.<var>@context</var> in the same order. Otherwise, set |verified| to
+`false` and skip to the last step.
```

Comment:
This change would make all existing secured documents invalid. Therefore, the name of the cryptosuite should be changed as well (to `eddsa-jcs-2024`, for example).

Reply:
It's designed not to invalidate existing proofs. Existing proofs should currently not contain a @context property, which means that this rule will have no effect on them. But maybe we can clarify in the rule that if there is no @context on the proof (which is equivalent to an empty set of contexts), no checks on the document context are performed.

Reply:
But if you have concerns regarding the logic of this or know of implementations that for some reason do add @context values to the proofs emitted please let us know, I can't guarantee that the solution considers everything there is to consider

Reply:
>Existing proofs should currently not contain a `@context` property, which means that this rule will have no effect on them. 

Oh, good to know.

>But maybe we can clarify in the rule that if there is no `@context` on the proof (which is equivalent to an empty set of contexts), no checks on the document context are performed.

Yes, I think this can be clarified. Currently it sounds like `@context` is required: https://github.com/w3c/vc-di-eddsa/pull/79#discussion_r1587597825

I don't know of any implementations that add `@context` to `proofOptions`